### ISSUE TITLE
Confusing text in security considerations

### ISSUE THREAD

**Issue by slagesse-epic** [IHE/ITI.DSUBm#35]
**Section Number** *Identify the most specific section number the issue occurs (e.g. 4.1.2)*

2:3.110.11

**Issue** *Describe your issue. Don't write a book, but do include enough to indicate what you see as a problem.*

I am not sure what the following text means:

"The Resource Notification Broker might maintain a whitelist of acceptable senders for the rest-create/rest-update methods. "

**Proposed Change** *Propose a resolution to your issue (e.g., suggested new wording or description of a way to address the issue). The committee might simply accept your suggested text. Even if they don't, it gives a good sense of what you are looking for. Leaving this blank means you can't imagine how to resolve the issue, which makes it easier for the committee to admit they can't imagine how to resolve it either and leave it unresolved.*

Please rephrase and/or clarify. 

**Priority**:
- Low: Typo or other minor classification that an editor can manage. Requires no group discussion.

**Comment by gpavan19** [comment]
the concept is that the Resource Notification Broker can have policies for which only Subscribers being part of a specific list can perform subscriptions or unsubscriptions. 

we can rephrase it to clarify @scontento
